mesheryctl/pkg/utils/helpers.go

```diff
@@ -591,3 +591,24 @@ func ValidateURL(URL string) error {
 	}
 	return nil
 }
+
+// GetLatestStableReleaseTag fetch and return latest stable release tag
+func GetLatestStableReleaseTag() (string, error) {
```

Comment:
This function is neat! It would be best to have it in the helpers.go file as this would be reused in the future.

Maybe this small change in the function doc.
```suggestion
// GetLatestStableReleaseTag fetches and returns the latest release tag from GitHub
func GetLatestStableReleaseTag() (string, error) {
```